its/src/test/java/com/sonar/it/scanner/msbuild/ScannerMSBuildTest.java

```diff
@@ -583,6 +583,16 @@ public void testCSharpAllFlat() throws IOException {
     assertThat(getComponent("CSharpAllFlat:Common.cs")).isNotNull();
   }
 
+  @Test
```

Comment:
In this file, only this change is relevant. 

The rest is just whitespace and the introduction of an overload to be able to perform this test.